style.css

```diff
@@ -21,6 +21,75 @@ body {
   display: none;
 }
 
+.popup-buttons {
+  display: flex;
+}
+
+.popup-default {
+  display: none;
+}
+
+.show {
+  display: block;
+  position: fixed;
+  top: 10px;
+  bottom: 10px;
+  border-radius: 0.5rem;
+  background: white;
+  left: 2rem;
+  right: 2rem;
+  padding: 1rem;
+}
```

Comment:
- [ ] Kindly in the mobile version you can give it more width to hide the background like the Figma design.

| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/35535130/156490233-7d2b748b-b2e0-4999-8520-ee4dd8de919c.png) | ![image](https://user-images.githubusercontent.com/35535130/156490298-75ac604e-b5fd-45f8-9895-86b955cc1718.png) |

Reply:
added